specification/sql/resource-manager/Microsoft.Sql/preview/2019-06-01-preview/managedInstanceOperations.json

```diff
@@ -0,0 +1,404 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2019-06-01-preview",
+    "title": "SqlManagementClient",
+    "description": "The Azure SQL Database management API provides a RESTful set of web APIs that interact with Azure SQL Database services to manage your databases. The API enables users to create, retrieve, update, and delete databases, servers, and other entities."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/managedInstances/{managedInstanceName}/operations/{operationId}/cancel": {
+      "post": {
+        "tags": [
+          "ManagedInstanceOperations"
+        ],
+        "description": "Cancels the asynchronous operation on the managed instance.",
+        "operationId": "ManagedInstanceOperations_Cancel",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/ManagedInstanceNameParameter"
+          },
+          {
+            "name": "operationId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "format": "uuid"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The request for cancel has been executed successfully."
+          },
+          "default": {
```

Comment:
Same as the other comment. Use standard schema reference.

Reply:
Same as: https://github.com/Azure/azure-rest-api-specs/blob/master/specification/sql/resource-manager/Microsoft.Sql/preview/2017-10-01-preview/cancelOperations.json

Reply:
Please note that without referencing default azure schema, the exception thrown will RestException. With this exception, detailed error message may not show up when failure occurs. User may just see BadRequest etc but not the detailed error message.